parser/ast/misc.go

```diff
@@ -1864,6 +1865,15 @@ type HandleRange struct {
 	End   int64
 }
 
+type StatementScope int
+
+const (
+	NoneScope StatementScope = iota
```

Comment:
It is better to name it `StatementScopeNone`, which is friendly to auto-complete.